### PR TITLE
Add note on finding templates

### LINKED ISSUE #1: Note: first board isn't built from template
Thank you so much for building this @bemosior (and introducing me to RealtimeBoard 😍). I ran through your readme and found the following behavior: 

* After sign up, I was launched into a new "blank" board
* When searching for templates inside the board, I didn't find a way to edit the template
* I then jumped back to my dashboard and found clicking "Blank board" showed me available templates
* I then searched and found yours

Given that behavior, I might offer a heads up in the readme for new users since it took me a little bit to realize I was in the wrong area. 

For reference, the search dialogue pops it right up! 

screen shot 2018-10-18 at 10 54 14 pm

Thank you again for this awesome tutorial and setup! You rock 🤘

### PR DESCRIPTION
**Pull request by mbbroberg**
Fixes #1 

Feel free to [make a suggestion](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change) (such a cool new feature!) if the language isn't exactly what you'd like. 

Thanks again for this ❤️ - finding it was very timely for me 😄

### CODE REVIEW

**Comment by mbbroberg** [comment]
Note that my original patch was wordy so I updated it (and rebased down to a single commit).

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The canvas has much room for improvement. Let's talk!
 
 1. Open an issue here, describing what you're aiming to improve (we'll have some helpful discussion)
 2. Register an account on [RealtimeBoard](https://realtimeboard.com)
-3. Create a new board
+3. Create a new board (Note: this must be [from the dashboard](https://realtimeboard.com/app/dashboard/) to use a template)
```
Comment (bemosior) [inline README.md]: What do you think about this wording? It puts the helpful link you added up-front, so they are tempted to just click it and get moving.
```suggestion
3. Open [the dashboard](https://realtimeboard.com/app/dashboard/) and create a new board
```

Comment (mbbroberg) [reply README.md]: It's much more concise. LGTM 👍

Comment (bemosior) [reply README.md]: Thank you!

**Comment by bemosior** [comment]
Thanks again for making this better! Really appreciate your help!

**Pull request merged**